modules/azurerm_netapp/variables.tf

```diff
@@ -1,12 +1,6 @@
 # Copyright © 2020-2023, SAS Institute Inc., Cary, NC, USA. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-variable create_netapp {
-  description = "Boolean flag to create Azure NetApp Files"
-  type        = bool
-  default     = false
-}
-
 variable "prefix" {
```

Comment:
No longer needed? This is unclear.

Reply:
Variable was specified in the `variables.tf` file but was not used in `main.tf`, hence tflint throws `terraform_unused_declarations` warning. Removing this does not affect the functionality.